modules/internal/ChapelStandard.chpl

```diff
@@ -70,7 +65,6 @@ module ChapelStandard {
 
   // Standard modules.
   use Assert;
-  use HaltWrappers only ;
```

Comment:
There's a bunch of modules that call the halt wrappers, I'm surprised you didn't need more uses of this.

Reply:
I believe the reason we get away with this is because the other cases use fully qualified references to HaltWrapper routines, and once _some_ module uses it, they see `HaltWrappers` in the global namespace.  If so, this is essentially issue #13523.  Here I took the approach of only adding the `use`s that were required to keep testing working, but with additional effort, I could try to grep all the places that use such modules and add uses there too (for HaltWrappers it would be easy since all uses are fully-qualified... for other cases I was worried it would be more work, which is why I didn't take it on... ).

Reply:
I see a few unqualified boundsCheckHalt calls in ChapelRange and all uses of unimplementedFeatureHalt. I guess I don't understand how those worked even before this change.